Rename -webkit-line-clamp to line-clamp (#37953)
mdn/content: files/en-us/web/css/line-clamp/index.md

---
title: line-clamp
slug: Web/CSS/line-clamp
page-type: css-property
browser-compat: css.properties.line-clamp
---

{{CSSRef}}

The **`line-clamp`** [CSS](/en-US/docs/Web/CSS) property allows limiting of the contents of a {{Glossary("block")}} to the specified number of lines.

> For legacy support with a vendor-prefix as `-webkit-line-clamp`, it only works in combination with the {{cssxref("display")}} property set to `-webkit-box` or `-webkit-inline-box` and the {{cssxref("box-orient", "-webkit-box-orient")}} property set to `vertical`. Despite them being deprecated, the co-dependency of these three properties is fully specified behavior and will continue to be supported.

Reviewer: ```suggestion
> [!NOTE]
> For legacy support, the vendor-prefixed `-webkit-line-clamp` property only works in combination with the {{cssxref("display")}} property set to `-webkit-box` or `-webkit-inline-box` and the {{cssxref("box-orient", "-webkit-box-orient")}} property set to `vertical`. Despite these prefixed properties being deprecated, the co-dependency of these three properties is a fully specified behavior and will continue to be supported.
```

> Currently this property is only supported with a vendor-prefix as `-webkit-line-clamp`. And only works in combination with the {{cssxref("display")}} property set to `-webkit-box` or `-webkit-inline-box` and the {{cssxref("box-orient", "-webkit-box-orient")}} property set to `vertical`. Despite them being deprecated, the co-dependency of these three properties is fully specified behavior and will continue to be supported.

Author: ```suggestion
For legacy support with a vendor-prefix as `-webkit-line-clamp`, it only works in combination with the {{cssxref("display")}} property set to `-webkit-box` or `-webkit-inline-box` and the {{cssxref("box-orient", "-webkit-box-orient")}} property set to `vertical`. Despite them being deprecated, the co-dependency of these three properties is fully specified behavior and will continue to be supported.
```
